### ISSUE TITLE
Get Key failed: Given final block not properly padded. Such issues can arise if a bad key is used during decryption.

### ISSUE THREAD

**Issue by abdullah432** [lohanidamodar/flutter_automation#5]
I'm getting the following issue after generating keys for an app. **Note:** I used it before and it was working fine

FAILURE: Build failed with an exception.                                
                                                                        
* What went wrong:                                                      
Execution failed for task ':app:signReleaseBundle'.                     
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Failed to read key everythink from store "/Users/.../..../..../keys/keystore.jks": Get Key failed: Given final block not properly padded. Such issues can arise if a bad key is used during decryption.

**Comment by OmarYehiaDev** [comment]
Hi there!
I'm facing the same problem. did you solve it or what?

**Comment by abdullah432** [comment]
@OmarYehiaDev Yes, It suddenly starts working after spending a lot of time. I'm still not sure what was the reason. Try flutter clean, restart the system. May it help

**Comment by OmarYehiaDev** [comment]
Oh okay
I'll try `flutter clean` and i hope it works

**Comment by OmarYehiaDev** [comment]
It didn't work so I undid everything and I followed this [article](https://medium.com/@psyanite/how-to-sign-and-release-your-flutter-app-ed5e9531c2ac[](url)) and it worked for me
